.github/workflows/test.yml

```diff
@@ -12,8 +12,20 @@ on:
       - 'releases/*'
 
 jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
+        env:
+          SHELLCHECK_OPTS: -x
```

Comment:
What's the best practice for option here? I saw some other projects using "-e SC1091" instead of "-x"

Reply:
-x allows shellcheck to also consider other files as well that may be sourced and the functions of those are called.
see shellcheck --help 
```
-x                  --external-sources         Allow 'source' outside of FILES
```

Reply:
I would only disable checks on a case by case basis. disabling for the entire project is only a ticking time bomb :D